scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
If this script is run from a directory that belongs to that repository, then it will run the hook from that repo. Otherwise, the config is set only locally for this repo (since it's missing the global flag).